python/triton/language/semantic.py

```diff
@@ -642,10 +656,12 @@ def bitcast(input: tl.tensor, dst_ty: tl.dtype, builder: ir.builder) -> tl.tenso
     return tl.tensor(builder.create_bitcast(input.handle, dst_ty.to_ir(builder)), dst_ty)
 
 
-def cast(input: tl.tensor, dst_ty: tl.dtype, builder: ir.builder) -> tl.tensor:
+def cast(input: tl.tensor, dst_ty: tl.dtype, builder: ir.builder, fp_downcast_rounding: str = None) -> tl.tensor:
```

Comment:
If the user asks for an unsupported rounding mode, AIUI the compiler currently crashes.  That's not very user-friendly -- for example, I doubt that will give a Python backtrace?

Seems like we should do the error checking for supported conversions in here, and produce user-friendly errors?

Reply:
(Or is the full matrix of rounding modes + src-type + dst-type now supported?)